MetriCam2.Controls/CameraExplorerControl.cs

```diff
@@ -67,8 +67,6 @@ public ListView.ListViewItemCollection SelectedCameras
 
         #region Private Fields
         private static CameraManagement cameraManagement;
-        private static readonly int IMAGE_SIZE_SMALL = 32;
```

Comment:
This might still be required. The default `Icon` c'tor returns the smallest available resolution.
If you want another one you have to explicitly request it.